background/redux-slices/assets.ts

```diff
@@ -258,13 +258,30 @@ export const transferAsset = createBackgroundAsyncThunk(
           ERC20_INTERFACE,
           provider
         )
-        let transactionDetails = await token.populateTransaction.transfer(toAddress, assetAmount.amount)
-        toAddress = transactionDetails.to? transactionDetails.to : ""
-        data = transactionDetails.data? transactionDetails.data : ""
-        assetAmount = {
-          asset: assetAmount.asset,
-          amount: BigInt(0),
+        let fromShard = getShardFromAddress(fromAddress)
+        let toShard = getShardFromAddress(toAddress)
+        if(fromShard != toShard) { // ERC20x cross-chain transfer
+          let minerTip = BigInt(1000000000) * BigInt(calcEtxFeeMultiplier(fromShard, toShard))
```

Comment:
Is it possible to not have hard coded magic numbers here?

Reply:
https://github.com/PelagusWallet/pelagus-extension/issues/7
Still an open issue

Reply:
Gwei